src/renderer/zbay/transit.js

```diff
@@ -237,9 +237,20 @@ export const packMemo = async (message, typingIndicato) => {
       )
       break
     default:
-      msgData = Buffer.alloc(MESSAGE_SIZE)
+      console.log("TRANSIT-zcash-onion", zcashAddressProp, onionAddressProp)
```

Comment:
In this `default` clause we are packing memo for message types: 
`BASIC` - which si what you wanted
`TRANSFER`
`AFFILIATE`
`CONNECTION_ESTABLISHED`

These other 3 types are either not used, or broken by the change you've made.

Reply:
In other words - it's better to add a specific case for `BASIC` here

Reply:
Also it wouldn't hurt to convert transit.js file to typesript.